pkg/tsdb/v3iotsdb_integration_test.go

```diff
@@ -619,6 +619,139 @@ func TestDeleteTSDB(t *testing.T) {
 	}
 }
 
+func TestDeleteTable(t *testing.T) {
+	v3ioConfig, err := tsdbtest.LoadV3ioConfig()
+	if err != nil {
+		t.Fatalf("unable to load configuration. Error: %v", err)
+	}
+
+	testCases := []struct {
+		desc         string
+		deleteFrom   int64
+		deleteTo     int64
+		deleteAll	 bool
+		ignorErrors  bool
+		data         []tsdbtest.DataPoint
+		expected     []tsdbtest.DataPoint
+		ignoreReason string
+		expectFail   bool
+	}{
+		{desc: "Should delete all table",
+			deleteFrom:     0,
+			deleteTo:       1599999999999,
+			ignorErrors:	true,
+			data:     		[]tsdbtest.DataPoint{{Time: 1538580053000, Value: 222.2},
```

Comment:
It would be nice to make the timestamps human readable (at least by adding comment)
Examples & Alternatives:
1. Absolute time
```
t := time.Now()
// Round the test time down to the closest hour to get predictable and consistent results
currentRoundedTimeNano := time.Date(t.Year(), t.Month(), t.Day(), t.Hour(), 0, 0, 0, t.Location()).UnixNano()
```
2. Use duration API: `now-2d`, etc.

Reply:
it should be absolut, not relative (like now-...)